Pollo/Views/Cards/MCResultCell.swift

```diff
@@ -163,13 +159,13 @@ class MCResultCell: UICollectionViewCell {
         
         numSelectedLabel.snp.makeConstraints { make in
             make.top.equalToSuperview().inset(numSelectedLabelTopPadding)
-            make.trailing.equalToSuperview().inset(numSelectedLabelTrailingPadding)
-            make.width.equalTo(numSelectedLabelWidth)
+            make.leading.equalTo(containerView.snp.trailing)
+            make.trailing.equalToSuperview()
```

Comment:
Shouldn't there be some degree of margin between the label and the leading and trailing views? Like at least 4 pixels on either side just to take care of the case where there were 4 digits and it was squashed against the containerView?